components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -268,8 +268,6 @@ export default class GroupsList extends React.PureComponent<Props, State> {
         let {searchString} = this.state;
 
         const newState = {...this.state};
-        delete newState.page;
```

Comment:
This isn't actually changing anything. When we deleted these items from `newState` before before passing it into `this.setState`, we were just telling React to leave them as-is which is what we're now still doing.